web-ui/src/main/resources/catalog/js/admin/HarvestController.js

```diff
@@ -28,13 +28,13 @@
         defaultTab: 'harvest-settings',
         tabs:
             [{
-              type: 'harvest-settings',
```

Comment:
FYI, the type should be the html template name. In that case, changing the type here just breaks tabs !